doc/source/whatsnew/v2.1.0.rst

```diff
@@ -84,10 +85,8 @@ Other enhancements
 - Improve error message when having incompatible columns using :meth:`DataFrame.merge` (:issue:`51861`)
 - Added to the escape mode "latex-math" preserving without escaping all characters between "\(" and "\)" in formatter (:issue:`51903`)
 - Improved error message when creating a DataFrame with empty data (0 rows), no index and an incorrect number of columns. (:issue:`52084`)
-- :meth:`DataFrame.applymap` now uses the :meth:`~api.extensions.ExtensionArray.map` method of underlying :class:`api.extensions.ExtensionArray` instances (:issue:`52219`)
```

Comment:
why remove this one?

Reply:
Looking at the output from `pre-commit run sort-whatsnew-items --all-files` and the CI tests, I thought that was the one that I needed to remove. Removing it also greened up the above command when I ran it locally. 

I can add it back in the appropriate place